### PR TITLE
modified to program and application (#4)

### PR DESCRIPTION
**Pull request by ubc-tuehoang**
* update conference to program

* updated paper to application

* modified paper to application for json files

* updated message

* updated help log for paper

* updated more help content fixes

---------

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -10,7 +10,6 @@ HOTCRP_CONTACT_NAME=Chairs
 HOTCRP_PAPER_SITE=
 
 
-
```
Comment (xcompass) [inline .env.default]: Could you remove this change? I would like minimal changes for the PR. Will be easier for merging in the future.

File: src/help/h_chairsguide.php [hunk 2]
```diff
@@ -116,10 +116,7 @@ static function print_assignments(HelpRenderer $hth, $gj) {
 
         } else if ($gj->itemid === 5) {
             echo "<li><p><strong>", $hth->setting_link("Prepare tracks (optional).", "track"),
-  "</strong> Tracks give chairs fine-grained control over PC
-  members’ access rights for individual papers. Example situations calling for
-  tracks include external review committees, PC-paper review committees, and
-  multi-track conferences.</li>\n";
+  "</strong> Tracks give chairs fine-grained control over PC members’ access rights for individual papers. Example situations calling for tracks include external review committees, PC-paper review committees, and multi-track programs.</li>\n";
```
Comment (xcompass) [inline src/help/h_chairsguide.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_chairsguide.php [hunk 3]
```diff
@@ -165,20 +162,12 @@ static function print_assignments(HelpRenderer $hth, $gj) {
 
     static function print_chair_conflicts(HelpRenderer $hth) {
         echo $hth->subhead("Chair conflicts");
-        echo "<p>Chairs and system administrators can access any information stored in the
-conference system, including reviewer identities for conflicted papers.
-It is easiest to simply accept such conflicts as a fact of life. Chairs
-who can’t handle conflicts fairly shouldn’t be chairs. However, HotCRP
-does offer other mechanisms for conflicted reviews.</p>
+        echo "<p>Chairs and system administrators can access any information stored in the program system, including reviewer identities for conflicted papers. It is easiest to simply accept such conflicts as a fact of life. Chairs who can’t handle conflicts fairly shouldn’t be chairs. However, HotCRP does offer other mechanisms for conflicted reviews.</p>
```
Comment (xcompass) [inline src/help/h_chairsguide.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_chairsguide.php [hunk 4]
```diff
@@ -165,20 +162,12 @@ static function print_assignments(HelpRenderer $hth, $gj) {
...
 
 <p>First, each paper can be assigned a <em>paper administrator</em>: a PC member
 who manages that paper’s reviewing and discussion. Use the left-hand side of a ",
 $hth->hotlink("submission’s assignment page", "assign"), " to enter its administrator. (You may need to
-“Override conflicts” to access the assignment page.)
-Paper administrators have full privilege to assign and view reviews for their
-papers, and can, for example, use the autoassignment tool, but they cannot change
-conference settings. When a paper
-has an administrator, chair conflicts cannot be overridden.</p>
+“Override conflicts” to access the assignment page.) Paper administrators have full privilege to assign and view reviews for their papers, and can, for example, use the autoassignment tool, but they cannot change program settings. When a paper has an administrator, chair conflicts cannot be overridden.</p>
```
Comment (xcompass) [inline src/help/h_chairsguide.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_chairsguide.php [hunk 5]
```diff
@@ -291,9 +280,8 @@ static function print_atmeeting(HelpRenderer $hth, $gj) {
   " to expose decisions to PC members if desired.</p></li>\n";
 
         } else if ($gj->itemid === 4) {
-            echo "<li><p><strong>Shepherding (optional).</strong> If your conference uses
-  shepherding for accepted papers, you can assign shepherds either ",
-  $hth->hotlink("paper by paper", "paper"), " or ", $hth->hotlink("automatically", "autoassign", "t=accepted"), ".</p></li>\n";
+            echo "<li><p><strong>Shepherding (optional).</strong> If your program uses shepherding for accepted applications, you can assign shepherds either ",
```
Comment (xcompass) [inline src/help/h_chairsguide.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_developer.php [hunk 6]
```diff
@@ -60,8 +60,7 @@ function print_usage() {
     }
 
     function print_settings() {
-        echo "<p>The chair-only <code>api/settings</code> endpoint accesses
-conference settings in ",
+        echo "<p>The chair-only <code>api/settings</code> endpoint accesses program settings in ",
     $this->hth->hotlink("JSON format", "help", ["t" => "jsonsettings"]) . ".
```
Comment (xcompass) [inline src/help/h_developer.php:65]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_developer.php [hunk 7]
```diff
@@ -91,8 +90,7 @@ function print_settings() {
     }
 
     function print_submissions() {
-        echo "<p>The <code>api/paper</code> endpoint accesses conference
-submissions. GET calls return paper information; use
+        echo "<p>The <code>api/paper</code> endpoint accesses program submissions. GET calls return paper information; use
```
Comment (xcompass) [inline src/help/h_developer.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_jsonsettings.php [hunk 8]
```diff
@@ -19,10 +19,7 @@ function __construct(HelpRenderer $hth) {
     function print() {
         echo "<p>With ",
             $this->hth->setting_group_link("Settings &gt; Advanced", "json"),
-            ", administrators can configure site operation by modifying a JSON
-specification. Advanced users can copy settings from one conference to
-another and configure aspects of the site not accessible via the normal
-settings UI.</p>";
+            ", administrators can configure site operation by modifying a JSON specification. Advanced users can copy settings from one program to another and configure aspects of the site not accessible via the normal settings UI.</p>";
```
Comment (xcompass) [inline src/help/h_jsonsettings.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_jsonsettings.php [hunk 9]
```diff
@@ -117,11 +114,9 @@ class=\"settings-jpath\">decision</code> is an array with one entry per
 }</code></pre></li>
 
 
-<li><p><strong>Copying settings between conferences.</strong> Beware of IDs
-when copying settings between conferences. Unless you are careful, the IDs
-used in one conference may differ from the IDs in another. Consider removing
-the IDs from a conference’s JSON settings before uploading those settings to
-another conference.</p></li>
+<li><p><strong>Copying settings between programs.</strong> Beware of IDs
```
Comment (xcompass) [inline src/help/h_jsonsettings.php:120]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_tags.php [hunk 10]
```diff
@@ -22,10 +22,7 @@ function print_intro() {
             $conflictmsg = " and conflicted PC members";
         }
 
-        echo "<p>PC members and administrators can attach tag names to papers.
-It’s easy to change tags and to list all papers with a given tag,
-and <em>ordered</em> tags preserve a particular paper order.
-Tags also affect color highlighting in paper lists.</p>
+        echo "<p>PC members and administrators can attach tag names to applicaitons. It’s easy to change tags and to list all applications with a given tag, and <em>ordered</em> tags preserve a particular application order. Tags also affect color highlighting in application lists.</p>
```
Comment (xcompass) [inline src/help/h_tags.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/help/h_tags.php [hunk 11]
```diff
@@ -51,18 +48,17 @@ function print_finding() {
...
     "</div></div></div></div><hr class=\"c\">
 
-<p>To find all papers with tag “#discuss”:</p>
+<p>To find all applications with tag “#discuss”:</p>
 
 <div class=\"mb-p\">", $hth->search_form("#discuss"), "</div>
 
-<p>You can also search with “", $hth->search_link("show:tags"), "” to see each
-paper’s tags, or “", $hth->search_link("show:#tagname"), "” to see a particular tag
+<p>You can also search with “", $hth->search_link("show:tags"), "” to see each application’s tags, or “", $hth->search_link("show:#tagname"), "” to see a particular tag
```
Comment (xcompass) [inline src/help/h_tags.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/pages/p_home.php [hunk 12]
```diff
@@ -179,10 +179,15 @@ function print_message() {
     }
 
     function print_welcome() {
+        echo '<div id="motivationalQuotes">
```
Comment (xcompass) [inline src/pages/p_home.php]: what about this change?

File: src/pages/p_mergeaccounts.php [hunk 13]
```diff
@@ -91,10 +91,7 @@ private function print() {
         $this->qreq->print_header("Merge accounts", "mergeaccounts");
 
         echo '<div class="w-text"><p>',
-            'You may have multiple accounts registered; perhaps you were asked to review
-papers using different email addresses. This form will allow you to transfer information
-between accounts, including authorship, reviews, and PC status. (Note that the
-transfer will only affect information currently stored in this conference.)</p></div>';
+            'You may have multiple accounts registered; perhaps you were asked to review applications using different email addresses. This form will allow you to transfer information between accounts, including authorship, reviews, and PC status. (Note that the transfer will only affect information currently stored in this program.)</p></div>';
```
Comment (xcompass) [inline src/pages/p_mergeaccounts.php]: Could you not change the line breaks but just the text? again easier for future merge.

File: src/userstatus.php [hunk 14]
```diff
@@ -1643,8 +1643,7 @@ static function print_topics(UserStatus $us) {
             return;
         }
         $us->cs()->add_section_class("w-text fx1")->print_start_section("Topic interests");
-        echo '<p>Please indicate your interest in reviewing papers on these conference
-topics. We use this information to help match papers to reviewers.</p>',
+        echo '<p>Please indicate your interest in reviewing applications on these program topics. We use this information to help match papers to reviewers.</p>',
```
Comment (xcompass) [inline src/userstatus.php]: Could you not change the line breaks but just the text? again easier for future merge.

**Review by xcompass — CHANGES_REQUESTED** [review]
I also remember you said you are going to make variables for changing those texts. Did you give it up?

File: src/pages/p_home.php (hunk 12 above)
Comment (ghost) [reply src/pages/p_home.php]: It was for fun - I will remove it.  😀

File: src/userstatus.php (hunk 14 above)
Comment (xcompass) [reply src/userstatus.php]: Just a reminder to change this.

File: src/help/h_chairsguide.php [hunk 15]
```diff
@@ -89,9 +89,9 @@ static function print_assignments(HelpRenderer $hth, $gj) {
             echo "</ol>\n\n";
 
         } else if ($gj->itemid === 1) {
-            echo "<li><p>Consider checking ", $hth->search_link("the papers", ["q" => "", "t" => "all"]),
+            echo "<li><p>Consider checking ", $hth->search_link("the apps", ["q" => "", "t" => "all"]),
```
Comment (xcompass) [inline src/help/h_chairsguide.php]: Could you change all `app` and `apps` to `application` and `applications`? The wording apps may confuse applicants.

**Review by xcompass — CHANGES_REQUESTED** [review]

**Comment by xcompass** [comment]
Thanks for the update. Almost there.

**Comment by xcompass** [comment]
Looks like some of the text in the tests are not updated causing the tests failing. Could you check? @tuehoang

**Pull request merged**